vispy/util/svg/color.py

```diff
@@ -155,12 +159,11 @@
     "yellowgreen":          (154, 205,  50)}
 
 
-_NUMERALS = '0123456789abcdefABCDEF'
-
-
-_HEXDEC = {v: int(v, 16)
-           for v in (x + y for x in _NUMERALS for y in _NUMERALS)}
-
+_HEXDEC = {}
+for x, y in combinations(hexdigits, 2):
+    v = x + y
+    _HEXDEC[v] = int(v, 16)
```

Comment:
```dict([(v, int(v, 16)) for v in (x + y for x in _NUMERALS for y in _NUMERALS)])```

@Eric89GXL et al this would have been possible btw. in Python 2.6